### PR TITLE
Add nbresuse

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jtpio** [comment]
cc @krinsman I added you to the list of maintainers, let me know if it's ok for you (and we can check the remaining box in the first comment).

Also if you know of other maintainers, we can update the PR and add them to the list. Or update the feedstock later on.

**Comment by krinsman** [comment]
@jtpio Yes, that's fine with me!

File: recipes/nbresuse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - notebook
+    - psutil
```
Comment (marcelotrevisani) [inline recipes/nbresuse/meta.yaml]:
```suggestion
    - psutil
    - python >=3.5
```

File: recipes/nbresuse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  imports:
+    - nbresuse
+
+about:
+  home: https://github.com/yuvipanda/nbresuse
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Jupyter Notebook Extension for monitoring Resource Usage"
+
```
Comment (marcelotrevisani) [inline recipes/nbresuse/meta.yaml]:
```suggestion
```

**Comment by jtpio** [comment]
@marcelotrevisani thanks for the review!

CI is passing, this should be good to go now.

**Comment by npavlovikj** [comment]
The recipe LGTM - thank you for contributing @jtpio 🎉

**Comment by jtpio** [comment]
Thanks @npavlovikj!

**Comment by jtpio** [comment]
@npavlovikj the feedstock was created, but it looks like it's empty: https://github.com/conda-forge/nbresuse-feedstock

**Comment by npavlovikj** [comment]
@jtpio , looks like the feedstock is populated now. There was some issue with the token I think, but looks like it works now. Thank you for reporting this!

**Comment by jtpio** [comment]
Thanks, it looks good now!

**Pull request merged**